Libraries/LibC/netdb.cpp

```diff
@@ -48,6 +49,8 @@ static hostent __gethostbyaddr_buffer;
 static char __gethostbyaddr_name_buffer[512];
 static in_addr_t* __gethostbyaddr_address_list_buffer[2];
 
+static auto services_file = Core::File::construct("/etc/services");
```

Comment:
Please don't add something just to remove it in a subsequent commit unless it's necessary to support an ongoing change.